api/encoding/call.go

```diff
@@ -156,6 +157,30 @@ func (c *Call) HeaderNames() []string {
 	return names
 }
 
+
+// HeaderNamesIter returns an iterator over the names of user defined headers
+// provided with this request.
+func (c *Call) HeaderNamesIter() iter.Seq[string] {
```

Comment:
why not "All"?

Reply:
The `call` has many more things in the metadata. `All()` can be misleading